### PR TITLE
refactor(wizards): migrate components to composition API

### PR DESCRIPTION
**Pull request by kleinfreund**
Migrates all wizard-related components to composition API.

Signed-off-by: Philipp Rudloff <philipp.rudloff@konghq.com>

### CODE REVIEW

File: src/app/wizard/views/DataplaneUniversal.vue [hunk 1]
```diff
@@ -464,282 +466,255 @@ networking:
...
-
-          console.error(error)
-        })
-        .finally(() => {
-          this.scanComplete = true
-        })
-    },
-    compeleteDataPlaneSetup() {
-      this.$store.dispatch('updateSelectedMesh', this.validate.meshName)
+function compeleteDataPlaneSetup(): void {
```
Comment (johncowen) [inline src/app/wizard/views/DataplaneUniversal.vue:709]: Similar to the other place I saw this, just a spelling fix up

**Review by johncowen — COMMENTED** [review]
Thanks for working through this, I left a couple of very minor comments. I remember the last one like this had a little bug fix in there also. Is this one a straight conversion or was there anything particularly different from a logic standpoint here also?

If not, if we just wanna fix up those spellings and maybe add a clearInterval in here also then we should be good to go!

File: src/app/wizard/views/DataplaneKubernetes.vue [hunk 2]
```diff
@@ -447,191 +468,145 @@ networking:
...
-    },
+    console.error(err)
+  } finally {
+    isComplete.value = true
+  }
+}
 
-    $route() {
-      const step = this.$route.query.step
+function compeleteDataPlaneSetup(): void {
```
[the remark this answers is not in the document]
Comment (kleinfreund) [reply src/app/wizard/views/DataplaneKubernetes.vue]: Done.

File: src/app/wizard/views/DataplaneUniversal.vue (hunk 1 above)
Comment (kleinfreund) [reply src/app/wizard/views/DataplaneUniversal.vue]: Done.

File: src/app/wizard/components/EntityScanner.vue [hunk 3]
```diff
@@ -59,96 +66,97 @@
...
+    clearInterval(intervalId.value)
+  }
+})
+
+function runScanner() {
+  isRunning.value = true
+  isComplete.value = false
+
+  // setup the interval function
+  intervalId.value = window.setInterval(() => {
```
[the remark this answers is not in the document]
Comment (kleinfreund) [reply src/app/wizard/components/EntityScanner.vue:141]: Sure, I added a clear call right before creating a new interval.

**Comment by kleinfreund** [comment]
> I remember the last one like this had a little bug fix in there also. Is this one a straight conversion or was there anything particularly different from a logic standpoint here also?

Nope, not this time.

**Review by johncowen — APPROVED** [review]
Look great, thanks for doing this!

**Pull request merged**